recipes/gecos/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "1.2.0" %}
+{% set number = "0" %}
+
+package:
+  name: gecos
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/gecos/archive/v{{ version }}.tar.gz
+  sha256: 48fc10ae4d018531b21815bedb49b095b33e13394ec3a69266d28e7c7c4bc01e
+
+build:
+  number: {{ number }}
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py < 37]
+  skip: true  # [osx]
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - numpy >=1.13
+    - biotite >=0.21
+    - scikit-image >=0.17
+    - matplotlib >=3.0
```

Unless you need a PyQt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base >=3.0
```